README.md

```diff
@@ -14,28 +14,36 @@ Go 1.7+ is officially supported. Older versions might work but are not tested.
 
 ## Usage
 
+Start by creating a new client instance with the appropriate configuration options:
+
 ```go
-// Create the client
-c, err := statsd.New("127.0.0.1:8125",
+client, err := statsd.New("127.0.0.1:8125",
```

Comment:
```suggestion
client, err := statsd.New(
    "127.0.0.1:8125",
```

Reply:
I think it's fine the way it currently is. It's easier to see the separation between the address and the list of options